app/views/accounts/_pii.html.slim

```diff
@@ -1,41 +1,87 @@
     .col.col-11
       = t('headings.account.profile_info')
       = image_tag asset_url('user.svg'), width: 8, class: 'ml1'
     .col.col-1.right-align
       = image_tag asset_url('lock.svg'), width: 8, class: 'mr1'
-  .p2
-    .clearfix.mxn1
-      .sm-col.sm-col-4.px1.bold = t('account.index.full_name')
-      .sm-col.sm-col-8.px1.truncate #{decrypted_pii.first_name} #{decrypted_pii.last_name}
-
-  .p2.border-top
-    .clearfix.mxn1
-      .sm-col.sm-col-4.px1.bold = t('account.index.address')
-      .sm-col.sm-col-8.px1
-        | #{decrypted_pii.address1}<br>
-        - if decrypted_pii.address2.present?
-          | #{decrypted_pii.address2}<br>
-        | #{decrypted_pii.city}, #{decrypted_pii.state} #{decrypted_pii.zipcode}
-
-  .p2.border-top
-    .clearfix.mxn1
-      .sm-col.sm-col-4.px1.bold = t('account.index.dob')
-      .sm-col.sm-col-8.px1 #{decrypted_pii.dob.to_date.to_formatted_s(:long)}
-
-  .p2.border-top
-    .clearfix.mxn1
-      .sm-col.sm-col-4.px1.bold = t('account.index.ssn')
-      .sm-col.sm-col-8.px1 = '***-**-****'
-
-  .p2.border-top.border-bottom
-    .clearfix.mxn1
-      .sm-col.sm-col-4.px1.bold = t('account.index.phone')
-      .sm-col.sm-col-8.px1 = decrypted_pii.phone
-
-  .bg-gray-lighter.px2.py1.clearfix.fs-12p
-    .col.col-12.sm-col-6
-      = image_tag asset_url('lock.svg'), width: 8, class: 'mr1'
-      = t('account.security.text')
-    = link_to t('account.security.link'), MarketingSite.help_url
+  - if decrypted_pii.present? && !locked_for_session
+    .p2
+      .clearfix.mxn1
+        .sm-col.sm-col-4.px1.bold = t('account.index.full_name')
+        .sm-col.sm-col-8.px1.truncate #{decrypted_pii.first_name} #{decrypted_pii.last_name}
+
+    .p2.border-top
+      .clearfix.mxn1
+        .sm-col.sm-col-4.px1.bold = t('account.index.address')
+        .sm-col.sm-col-8.px1
+          | #{decrypted_pii.address1}<br>
+          - if decrypted_pii.address2.present?
+            | #{decrypted_pii.address2}<br>
+          | #{decrypted_pii.city}, #{decrypted_pii.state} #{decrypted_pii.zipcode}
+
+    .p2.border-top
+      .clearfix.mxn1
+        .sm-col.sm-col-4.px1.bold = t('account.index.dob')
+        .sm-col.sm-col-8.px1 #{decrypted_pii.dob.to_date.to_formatted_s(:long)}
+
+    .p2.border-top
+      .clearfix.mxn1
+        .sm-col.sm-col-4.px1.bold = t('account.index.ssn')
+        .sm-col.sm-col-8.px1 = '***-**-****'
+
+    .p2.border-top.border-bottom
+      .clearfix.mxn1
+        .sm-col.sm-col-4.px1.bold = t('account.index.phone')
+        .sm-col.sm-col-8.px1 = decrypted_pii.phone
+
+    .bg-gray-lighter.px2.py1.clearfix.fs-12p
+      .col.col-12.sm-col-6
+        = image_tag asset_url('lock.svg'), width: 8, class: 'mr1'
+        = t('account.security.text')
+      = link_to t('account.security.link'), MarketingSite.help_url
+  - else
+    .p2
+      .clearfix.mxn1
+        .sm-col.sm-col-4.px1.bold = t('account.index.full_name')
+        .sm-col.sm-col-8.px1.truncate = '***** **********'
```

Comment:
Sorry to clarify --- when I meant that we should combine into one template --- I didn't mean one big if/else

For accounts with obfuscated PII, we should supply a different object that has the same methods as `decrypted_pii` but the values come back obfuscated instead.

Let me know if you want to pair on this